src/haz3lschool/SchoolExercise.re

```diff
@@ -255,6 +255,7 @@ module F = (ExerciseEnv: ExerciseEnv) => {
     | 2 => YourTestsTesting
     | 3 => YourTestsValidation
     | 4 => YourImpl
+    | (-1) => HiddenTests
```

Comment:
I don't think this is not correct -- should be the inverse of idx_of_pos, which is already handled in the catchall case below

Reply:
Removed